test/px-data-grid-test.js

```diff
@@ -189,5 +208,60 @@ function runTests() {
         expect(grid._generatedColumns.length).to.be.eql(0);
       });
     });
+
+    describe('auto filter field tests', () => {
+
+      beforeEach(() => {
+        grid = fixture('grid-with-auto-filter');
+        grid.tableData = data;
+        grid.autoFilter = true;
+      });
+
+      it('should show only 1 row after filtering', (done) => {
+        Polymer.RenderStatus.afterNextRender(grid, () => {
```

Comment:
DRY: move this line and line before to `beforeEach`, remove copy-pasted stuff from all test cases inside this `describe` section.